conftest.py

```diff
@@ -138,7 +138,7 @@ def _image_dl_estimator_for_attack(attack, defended=False, **kwargs):
             raise ARTTestFixtureNotImplemented(
                 "no estimator available", image_dl_estimator_for_attack.__name__, framework, {"attack": attack}
             )
-        return classifier_tested[0]
+        return classifier_tested[0], sess
```

Comment:
Why is this change required? The returned `sess` does not seem to be used anywhere.

Reply:
ye it's rarely used indeed but the session is needed some times for Tensorflow models. That's a discrepancy that I hadn't seen before. The session is used in some test cases when `image_dl_estimator` is called directly, hence `image_dl_estimator_for_attack` which is just an augmented version of  `image_dl_estimator` by convention should have passed on that session too from the start. That was something I had overlooked so I just felt I should fix this now rather than later when someone complains why isn't it passed on :-)